wordpress-mobile/WordPress-Android
# Issue/10641 material activity log

Fixes #10641

This PR moves Activity Log screen to Material Theme and adds Dark Theme support. There are some code changes for cosmetic reasons.

While refactoring is not in the scope of this PR, since I touched those files, I cleaned up/optimized some of the redundant code as suggested by AS, so do not be alarmed by the size of the PR. I suggest turning on "Hide whitespace changes" option during the review.

There is a huge amount of PRs in this feature that covers all the screens within the app, so I'm requesting reviews from a wide variety of people.

![comparison_activitylog](https://user-images.githubusercontent.com/728822/71052059-1dfb9e00-20ff-11ea-8d0a-bf8fc56d5b59.png)

To test:
- Navigate to Activity Log screen and confirm it's visual and functional integrity in both Light and Dark themes. The theme can be changed from App Settings.

PR submission checklist:
